.github/workflows/liquidation-reconstitution.yml

```diff
@@ -17,3 +27,4 @@ jobs:
         docker-compose -f test/e2e/docker-compose-reconstitution.yml \
         --profile synpress up --build \
         --exit-code-from synpress
+      AGORIC_NET: ${{ inputs.network }}
```

Comment:
why not leave the other code as is? we can just set this condition to: 
`${{  inputs.network === 'emerynet' }}`